WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -41,8 +53,11 @@ class JetpackWindowManager: WindowManager {
         showAppUI(for: blog)
     }
 
-    // TODO: Add logic in here to trigger migration UI if needed
     private var shouldShowMigrationUI: Bool {
-        return FeatureFlag.contentMigration.enabled
+        return FeatureFlag.contentMigration.enabled && AccountHelper.isLoggedIn
```

Comment:
@alpavanoglu I'm curious if we should check `AccountHelper.hasBlogs` too.  Because we do the same check before we display the app UI.

```swift
        if AccountHelper.isLoggedIn && AccountHelper.hasBlogs {
            showAppUI(for: blog)
            return
        }
```

Reply:
I don't know really why we have that check there too. For some reason we log them out if they don't have any blogs. Maybe we can ask for others' input.

Reply:
I wonder if we still need this boolean variable? We only want to show the migration UI after the user goes through the data migration — i.e. in the completion block of the `DataMigrator`.